### PR TITLE
uik: add token rotation dialogs and controls

### PR DESCRIPTION
**Pull request by mastercactapus**
**Description:**
Adds token rotation dialogs and functionality, along with integration test for universal integration keys

### CODE REVIEW

File: web/src/app/services/UniversalKey/UniversalKeyPage.tsx [hunk 1]
```diff
@@ -1,16 +1,138 @@
...
+      keyID: props.keyID,
+      serviceID: props.serviceID,
+    },
+  })
+
+  // Redirect to the correct service if the key is not in the service
+  if (
+    q.data &&
+    q.data.integrationKey &&
+    q.data.integrationKey.serviceID !== props.serviceID
```
Comment (Forfold) [inline web/src/app/services/UniversalKey/UniversalKeyPage.tsx:57]: we should be able to simplify this:

```
if (q?.data?.integrationKey?.serviceID !== props.serviceID) {
```

**Review by Forfold — DISMISSED** [review]

**Comment by Forfold** [comment]
`make test` passes locally- rerunning on CI

**Review by allending313 — APPROVED** [review]

File: web/src/app/loading/components/LoadingButton.tsx [hunk 2]
```diff
@@ -34,7 +34,7 @@ const LoadingButton = (props: LoadingButtonProps): JSX.Element => {
         disabled={loading || disabled}
         type={noSubmit ? 'button' : 'submit'}
       >
-        {buttonText || (!attemptCount ? 'Confirm' : 'Retry')}
+        {!attemptCount ? buttonText || 'Confirm' : 'Retry'}
```
Comment (Forfold) [inline web/src/app/loading/components/LoadingButton.tsx:37]: This is not preferable moving forward as we want the button to say a specific text ALWAYS, and not have it change to "Retry" on the first click.

Comment (mastercactapus) [reply web/src/app/loading/components/LoadingButton.tsx:37]: This is handled by the `alert` prop on FormDialog -- once we have a token we set that prop to `true` which forces the button to always say `DONE`

Comment (mastercactapus) [reply web/src/app/loading/components/LoadingButton.tsx:37]: Changing this logic causes playwright tests to fail as other parts of the application count on `Retry` overriding the `buttonText` after the first attempt

**Review by Forfold — APPROVED** [review]

**Pull request merged**